Makefile

```diff
@@ -66,27 +71,44 @@ endif
 endif
 endif
 
+ifeq ($(grep -c .NETCoreApp,Version= bin/OpenRA.Utility.dll),0)
```

Comment:
This and the awkward evals below can be avoided by calling `utility.sh` instead of invoking it directly.